xmtp_id/src/utils/mod.rs

```diff
@@ -1,3 +1,11 @@
-pub mod time;
+use wasm_timer::{SystemTime, UNIX_EPOCH};
```

Comment:
wasm_timer uses normal `std` system_time libraries if on native, and the JS web_sys/js_sys shim if on web